app/schema.ts

```diff
@@ -116,6 +118,21 @@ export const PrivateCloudCreateRequestBodySchema = z.object({
   commonComponents: CommonComponentsInputSchema,
   golddrEnabled: z.preprocess(processBoolean, z.boolean()),
   isTest: z.preprocess(processBoolean, z.boolean()),
+  supportPhoneNumber: z
+    .string()
+    .nullable()
```

Comment:
Are `nullable` and `optional` still necessary given that these values are handled within the `refine` method?